src/main/kotlin/apply/domain/mission/judgmentItem/JudgmentItem.kt

```diff
@@ -0,0 +1,28 @@
+package apply.domain.mission.judgmentItem
+
+import support.domain.BaseEntity
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+
+@Entity
+class JudgmentItem(
+    @Column(nullable = false)
```

Comment:
r: 추가적인 제약 조건이 필요합니다. 마이그레이션 SQL에도 반영해 주세요.

```suggestion
    @Column(unique = true, nullable = false)
```